publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java

```diff
@@ -78,6 +78,12 @@ public void loadAd() {
     }
   }
 
+  public void displayAd(@NonNull String signalData) {
+    // FIXME: call notifyFor? Assuming publishers would pass a listener during the main SDK setup
```

Comment:
Could you explain more what you're expecting ?
This will call the onAdReceived of the listener. Semantically, ad is already received at this point.

Reply:
Removed. It doesn't make sense indeed.